packages/local-plugin/src/utils.ts

```diff
@@ -0,0 +1,48 @@
+const TEST_KEY = "__storage_test__";
+
+export const isLocalStorageAvailable = () => {
+  try {
+    if (typeof window.localStorage === "undefined") {
```

Comment:
Good point - though in that case it will be caught by the try/catch anyway. 

This is essentially inspired by https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Feature-detecting_localStorage - not sure if we want to include the error code checking as well...

Will make the change for readability anyway.